modules/deployment/trigger.tf

```diff
@@ -1,22 +1,22 @@
-resource "aws_cloudwatch_event_rule" "this" {
+resource "aws_cloudwatch_event_rule" "ecr-action" {
   name        = "${var.service_name}-ecr-trigger"
-  description = "Capture ECR push events."
+  description = "Capture ECR push and replication events."
 
   tags = merge(var.tags, {
     tf_module = basename(path.module)
   })
 
-  event_pattern = <<PATTERN
+  event_pattern = <<HEREDOC
 {
     "detail-type": [
-        "ECR Image Action"
+        "ECR Image Action", "ECR Replication Action"
```

Comment:
we could probably remove the `detail-type` from the matcher, but this is a nit pick